src/processTranslation.tsx

```diff
@@ -60,14 +69,28 @@ export const processTranslation = ({
     throw new Error(`Missing translation: "${key}" with suffix: "${suffix}"`);
   }
 
+  const {
+    context,
+    ns,
+    ...replaceableArgs
+  } = args;
+
   // Replace placeholders like `{{count}}` with values from `args`
-  const processed = Object.entries(args).reduce(
-    (v, [ name, value ]) => v.replace(
-      new RegExp(`{{\\s*${name}\\s*}}`, 'g'),
-      String(formatArgument(locale, value)),
-    ),
+  return Object.entries(replaceableArgs).reduce(
+    (v, [ name, value ]) => {
+      const regexp = new RegExp(`{{\\s*${name}\\s*}}`, 'g');
+
+      if (onError && !regexp.test(v)) {
+        onError(
```

Comment:
I'm still thinking what I said in the previous one.

"_you don't want to close the door for doing this with other types of errors, so you want to either have different functions (passed in an object) or pass the type of error in the callback (preferred)_"

`onError` is super generic, and when the callback gets called (as a user) you don't know exactly what it is.

Suggestion:
```
type: 
   onError: (errorType: 'REPLACING_WHATEVER' | 'TRANSLATION_NOT_FOUND' | 'SOMETHING_NEW', params: REPLACING_WHATEVER_PARAMS | TRANSLATION_NOT_FOUND_PARAMS | SOMETHING_NEW_PARAMS)
   
 and here:
onError(REPLACING_WHATEVER, { key, argumentName, value } 
```

Reply:
(just in casey,  my 'whatevers' are just to show the point, not actual naming suggestion)

Reply:
I got what you were saying now,
Nice, that makes sense.

I've added just one error type for now: `REPLACE_ARGUMENT_NOT_FOUND`.
This list could be easily expanded later if we'll need it.